public/javascripts/components/nav.js

```diff
@@ -61,7 +61,7 @@ window.ChubboChat.components.navbar = Vue.extend({
     onDashboard: function() {
       //dashboard: show menu icon or logout btn (based on css media queries)
       //landing: show dashboard shortcut or login btn (based on user login status)
-      if (this.$route.path === '/dashboard') {
+      if (this.$route.path === '/dashboard/survey' || '/dashboard/responses') {
```

Comment:
I think that this boolean will not evaluate properly. it should be 2 expressions.
a == b || a == c
also, can we just go:
`` return (a == b || a == c ) ``